examples/qtmegachatapi/SettingWindow.cpp

```diff
@@ -110,6 +119,27 @@ void SettingWindow::onPushNotificationSettingsUpdate()
     ui->pushNotifications->setEnabled(true);
 }
 
+void SettingWindow::onPresenceConfigUpdate()
+{
+    ui->presence->setEnabled(false);
+
+    delete mPresenceConfig;
```

Comment:
I think mPresenceConfig has to be deleted at SettingWindow destructor too

Reply:
You're right. Fixing...